src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -107,6 +126,41 @@ public void testCreateAppeal_throwsExceptionIfUnableToInsertData() {
         assertThrows(Exception.class, () -> appealService.saveAppeal(appeal, TEST_ERIC_ID));
     }
 
+    @Test
+    public void testCreateAppealChipsEnabled_returnsResourceId() throws Exception {
+
+        when(chipsConfiguration.isChipsEnabled()).thenReturn(true);
+        when(chipsConfiguration.getChipsRestServiceUrl()).thenReturn(TEST_CHIPS_URL);
+
+        final Appeal appeal = getValidAppeal();
+        appeal.setId(TEST_RESOURCE_ID);
+
+        when(appealRepository.insert(any(Appeal.class))).thenReturn(appeal);
+
+        assertEquals(TEST_RESOURCE_ID, appealService.saveAppeal(appeal, TEST_ERIC_ID));
+    }
+
+    @Test
+    public void testCreateAppealChipsEnabled_throwsExceptionIfChipsReturnsError() {
+
+        final ArgumentCaptor<Appeal> appealArgumentCaptor = ArgumentCaptor.forClass(Appeal.class);
+
+        when(chipsConfiguration.isChipsEnabled()).thenReturn(true);
+        when(chipsConfiguration.getChipsRestServiceUrl()).thenReturn(TEST_CHIPS_URL);
+
+        doThrow(ChipsServiceException.class).when(chipsRestClient).createContactInChips(any(ChipsContact.class), anyString());
```

Comment:
when(...).then(...) seems to be the preferred pattern

Reply:
I agree with this, and when(..).then(..) is more readable, but unfortunately this doesn't work for `void` return types